src/controllers/user.js

```diff
@@ -0,0 +1,59 @@
+import Model from '../models';
+
+const { Users } = Model;
+
+/**
+* Generic update profile
+
+* @function
+*
+* @param {request} request - express request object
+*
+* @return {Object} - callback that execute the controller
+*
+* @description allows users to update their profile
+*/
+const updateProfile = async (request) => {
```

Comment:
Please export the whole controller as an object and done in the other controllers

Reply:
Done.